meson.build

```diff
@@ -0,0 +1,17 @@
+project(
+  'flatpak-xdg-utils', 'c',
```

Comment:
I recommend always setting a minimum meson_version

Reply:
I searched in the docs for all the stuff I used here, and none specified "this was introduced in Meson x.y.z", so I didn't know what to use as a minimum. What do you recommend is a good minimum?

Reply:
I used the meson version in org.freedesktop.Sdk//1.6 (0.46.0), in case we ever want to upgrade flatpak-xdg-utils there.